.github/workflows/continuous-integration.yml

```diff
@@ -47,8 +52,9 @@ jobs:
       - run: yarn install --frozen-lockfile --non-interactive
 
       - name: Run tests
-        # Try yarn test:prod up to three times, if it fails and exit with the exit code from the last execution
```

Comment:
I today triggered several re runs and no one had a "operation cancelled" exit. So it could be that your fix helps. But I'd suggest to re add our retry mechanism, as we still have some failing builds from time to time